sb1-scope.json

```diff
@@ -59,7 +59,7 @@
         },
         {
           "enabled": true,
-          "file": "\\/sb1-nettsider-skjema\\/.*",
+          "file": "\\/sb1-nettsider-skjema\\/rest\\/.*",
```

Comment:
Når jeg diskuterte dette med @lonnelars i forrige uke så kom vi fram til at kun disse urlene trenger å ha unntak.
```
/sb1-nettsider-skjema/rest/${bank}/bm/signing/initiate
/sb1-nettsider-skjema/rest/${bank}/initiatesigning
/sb1-nettsider-skjema/rest/${bank}/sendnosign
/sb1-nettsider-skjema/rest/${bank}/send
```

Reply:
Det er ikke mulig å teste noe som helst av skjemaløsningen når hele `/rest/` er unntatt. F.eks. så hentes skjemadefinisjonen fra `/sb1-nettsider-skjema/rest/${bank}/id/${id}`

Reply:
Skjønner! Har dere full oversikt over endepunktene for innsending av skjema, så de 4 du lister opp dekker alt? @a504aa1

Reply:
Ja, vi har gått gjennom alle endepunktene. Disse fire skal dekke alt som kan plage rådgivere.

Det vil være mulig å åpne, fylle ut, laste opp vedlegg og lagre utkast, men ikke signere eller sende inn.

Reply:
Skjematjenesten brukes vel også av bl.a. SR til å sende data til automatisert behandling av roboter. Det dekkes også av disse endepunktene? @a504aa1

Reply:
Ja. Etter at brukeren har sendt inn skjemaet så ruter vi dataene til riktig mottaker.